### PR TITLE
Update README: reference, history, Cobrawap Community 

### PR DESCRIPTION
**Pull request by gulpgiulia**
Details:
(i) reference updated (from arxive to final doi); (ii) project history updated (from HBP-Wavescales to present status in EBRAINS); (iii) Cobrawap Community updated (from "Wavescales involved partners" to illustration of Core-Team, Support-Team and Scientific Partners, and list of involved people in alphabethical order).

++other minor changes (punctuations, slight reformulations, etc...)

### CODE REVIEW

**Comment by gulpgiulia** [comment]
see also PR#59 (that is on Update README as well)

File: README.rst [hunk 1]
```diff
@@ -64,31 +64,41 @@ To cite a specific version of Cobrawap please see version-specific DOIs at:
...
 License
 =======
 Cobrawap is open-source software and is licensed under the `GNU General Public License v3 <https://github.com/NeuralEnsemble/cobrawap/blob/master/LICENSE>`_.
 
 
-Involved partners
+The Cobrawap Community
 =================
-Cobrawap was originally developed in the context the `Human Brain Project <https://www.humanbrainproject.eu>`_ and is part of the `EBRAINS <https://www.ebrains.eu>`_ infrastructure. Partners involved in the collaboration and use case within the project (*WaveScalES*) are:
+Cobrawap is currently provided as a `tool <https://www.ebrains.eu/tools/cobrawap>`_ of the `EBRAINS <https://www.ebrains.eu>`_ infrastructure and included in the `EBRAINS-Italy <https://www.ebrains-italy.eu/>`_ initiative. Further details on fundings and resources are in the `Acknowledgments <https://github.com/NeuralEnsemble/cobrawap/blob/master/doc/source/acknowledgments.rst>`_ file in the doc session.
```
Comment (cosimolupo) [inline README.rst:78]: At the very end of the statement, there is a typo: "session" -> "section", or even better "folder".

Comment (gulpgiulia) [reply README.rst]: done. new commit added.

**Review by cosimolupo — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -64,31 +64,41 @@ To cite a specific version of Cobrawap please see version-specific DOIs at:
...
-- **European Laboratory for Non-Linear Spectroscopy (LENS), Firenze, Italy:** Anna Letizia Allegra Mascaro, Francesco Resta, Francesco Pavone.
+- **European Laboratory for Non-Linear Spectroscopy (LENS), Firenze, Italy**
 
-- **University of Milano (UniMi), Italy:** Andrea Pigorini, Thierry Nieus, Marcello Massimini
-
-- **Unité de Neurosciences, Neuroinformatics Group, CNRS, France:** Andrew Davison
+- **University of Milano (UniMi), Italy**
+  
+Other people involved (past and present):
+Anna Letizia Allegra Mascaro (LENS), Irene Bernava (INFN), Cristiano Capone (INFN), Alessandra Cardinale (INFN), Miguel Dasilva (IDIBAPS), Chiara De Luca (INFN), Gianluca Gaglioti (UniMi), Arnau Manasanch (IDIBAPS), Marcello Massimini (UniMi), Maurizio Mattia (ISS), Thierry Nieus (UniMi), Francesco S. Pavone (LENS), Andrea Pigorini (UniMi), Francesco Resta (LENS), Maria V. Sanchez-Vives (IDIBAPS).
```
Comment (mdenker) [inline README.rst:102]:
```suggestion

**Cobrawap Partnering Projects**:

Sleep Wave Analysis Visualization Engine (SWAVE): A data visualization tool that takes Cobrawap outputs and visualizes these to show dynamic wave-like activity patterens found in the data. Developed at: https://github.com/cilantroxiao/SWAVE
```

File: README.rst [hunk 3]
```diff
@@ -64,31 +64,41 @@ To cite a specific version of Cobrawap please see version-specific DOIs at:
...
 
 
 License
 =======
 Cobrawap is open-source software and is licensed under the `GNU General Public License v3 <https://github.com/NeuralEnsemble/cobrawap/blob/master/LICENSE>`_.
 
 
-Involved partners
+The Cobrawap Community
 =================
```
Comment (mdenker) [inline README.rst:77]:
```suggestion
=======================
```

File: README.rst [hunk 4]
```diff
@@ -33,9 +33,9 @@ Concept
 
 For researchers to be able to effectively reproduce results and build on each other's progress, it is important to not only make results openly accessible and to facilitate data sharing, but also to build the analysis workflows in a shareable and reusable manner.
 
-Making analysis scripts available alongside results and datasets is good. What is even better is to design the analysis workflows for specific research questions in such a manner that they are general and flexible enough to be actively reused in further research. In this way, the rigor of the analysis workflow is increased, and the design of the analysis process is simplified for the researcher
+Making analysis scripts available alongside results and datasets is good. What is even better is to design the analysis workflows for specific research questions in such a manner that they are general and flexible enough to be actively reused in further research. In this way, the rigor of the analysis workflow is increased, and the design of the analysis process is simplified for the researcher.
```
[the remark this answers is not in the document]
Comment (gulpgiulia) [reply README.rst:36]: Fine with me, I agree with your point, we can remove both paragraphs; check if @rgutzen is fine as well, I didn't want to overturn his formulation and communication objective

File: README.rst [hunk 5]
```diff
@@ -64,32 +59,45 @@ To cite a specific version of Cobrawap please see version-specific DOIs at:
...
 License
 =======
 Cobrawap is open-source software and is licensed under the `GNU General Public License v3 <https://github.com/NeuralEnsemble/cobrawap/blob/master/LICENSE>`_.
 
 
-Involved partners
-=================
-Cobrawap was originally developed in the context the `Human Brain Project <https://www.humanbrainproject.eu>`_ and is part of the `EBRAINS <https://www.ebrains.eu>`_ infrastructure. Partners involved in the collaboration and use case within the project (*WaveScalES*) are:
+The Cobrawap Community
+=======================
```
Comment (cosimolupo) [inline README.rst:72]:
```suggestion
======================
```

**Review by mdenker — APPROVED** [review]

**Pull request merged**